src/Detector/Hugo/HugoDetector.cs

```diff
@@ -155,23 +167,25 @@ private bool IsHugoApp(ISourceRepo sourceRepo, out string appDirectory)
         private bool IsHugoTomlFile(ISourceRepo sourceRepo, params string[] subPaths)
         {
             TomlTable tomlTable = null;
-
-            var relativeFilePath = Path.Combine(subPaths);
-
-            try
-            {
-                tomlTable = ParserHelper.ParseTomlFile(sourceRepo, relativeFilePath);
-            }
-            catch (FailedToParseFileException ex)
+            foreach (string subPath in subPaths)
```

Comment:
Aren't these multiple sub paths, like and example of this object could be `[ 'C:', 'repos', 'samplehugoapp', 'hugo.toml' ]`? So why are we looping through it here instead of combining it first?

Maybe I'm misreading something though. Also, as far as i can tell, we never pass in more than 2 params, so doesn't seems like its really even being used.

Reply:
Thanks for the suggestion! `IsHugoTomlFile` is being called from [this line](https://github.com/microsoft/Oryx/blob/main/src/Detector/Hugo/HugoDetector.cs#LL70C17-L70C17). And seems like the object in the past is just `config.toml`. But as you said, we actually never pass more than 2 parameters (in fact, many of the constants are also not being used in many detector's platform constant files). I'll try removing it and see if it breaks anything here.